CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ bet we will fix some bugs and make a world even a better place.
 
 ### Fixes
 
-- `GitRepo.dirty` does not report on nested empty directories ([#3196]).
+- `GitRepo.dirty` does not report on nested empty directories ([#3196][]).
```

Comment:
just so I know - why is it (explicit empty `[]`) needed?

We would need to have `tools/link_issues_CHANGELOG` adjusted for it as well then I believe

Reply:
commit message unfortunately is not informative.  AFAIK it was a valid markdown, e.g. according to examples at https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
It may be valid, but then it is not supported by pandoc. On conversion to RST all that gets stripped.